lib/modules/subredditManager.js

```diff
@@ -438,6 +438,8 @@ function showSubredditGroupDropdown(obj) {
 }
 
 function _showSubredditGroupDropdown(obj, subreddits, suffix) {
+	// Updates source object referenced on the 'click' events
+	subredditGroupDropdownRefItem = obj;
```

Comment:
I don't really like this, but the existing implementation already has a ton of global state so we can't really do better